### ISSUE TITLE
Is the verification loss only for better training of the feature extraction model?

### ISSUE THREAD

**Issue by NeoYY** [layumi/Person_reID_baseline_pytorch#324]
I was previously expecting that the verification part is also part of the inference and testing, but I realized you still only used the ft_net for the feature extraction and then compute the distance then ranked and selected. In such a circumstance, is it true that the verification loss is only helping the faster convergence of the ft_net? If so, is there any possibility to use the verification model in the inference? Thanks!

**Comment by layumi** [comment]
Yes. The verification loss is for training a better model. You may refer to one of my early paper. 
https://publons.com/publon/20522872/ 

But it is also possible to be used as post-processing to finetune the final ranking list. Some paper also argues that verification can help the re-ranking top-K result with better retrieval performance.
